client/web/src/search/panels/RepositoriesPanel.tsx

```diff
@@ -142,19 +148,81 @@ export const RepositoriesPanel: React.FunctionComponent<React.PropsWithChildren<
                     ))}
                 </ul>
             )}
-            {searchEventLogs?.pageInfo.hasNextPage && (
+                        {searchEventLogs?.pageInfo.hasNextPage && (
+                // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
```

Comment:
The eslint exception should not be needed here.

```suggestion
                {searchEventLogs?.pageInfo.hasNextPage && (
```